src/containers/ShoppingList/index.js

```diff
@@ -0,0 +1,110 @@
+/**
+ *
+ * ShoppingList
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Layout, List, Icon, Checkbox } from 'antd';
+import './ShoppingList.css';
+import 'antd/dist/antd.css';
+const { Content } = Layout;
+
+const drinks = [
```

Comment:
Try to work with data as similar as the production one. I suppose you're going to work with a single object with properties like drinks, meat, etc…

Reply:
Sure. I made this just as a view to see how it would look. I'm working right now on rendering this with the data provided!